templates/download/server/thank-you.html

```diff
@@ -13,11 +13,11 @@
 </noscript>
 {% endif %}
 
-<section class="p-strip is-bordered u-no-padding--top u-no-padding--bottom">
+<section class="p-strip u-no-padding--top u-no-padding--bottom" style="overflow: visible;">
   <div class="p-strip--suru-topped">
```

Comment:
When I subscribe to Ubuntu Server weekly news, this is what I see:

![image](https://user-images.githubusercontent.com/19801137/69327404-65e4ed80-0c45-11ea-82a2-040628d7b994.png)

Ideally this notification would not be on a new page, it would just appear below the button on the previous page. But if we do go to a new page, the notification should be below the Suru topper, not above it.

Reply:
Done your oyster world version